web/src/pages/apd/proposed-budget/ProposedBudgetReadOnly.js

```diff
@@ -1,19 +1,22 @@
+import PropTypes from 'prop-types';
 import React from 'react';
+import { selectApdType } from '../../../redux/selectors/apd.selectors';
+import { connect } from 'react-redux';
 
 import CombinedActivityCosts from './CombinedActivityCosts';
 import BudgetSummary from './BudgetSummary';
 import IncentivePayments from './IncentivePayments';
 import QuarterlyBudgetSummary from './QuarterlyBudgetSummary';
 
-const ProposedBudget = () => (
+const ProposedBudget = ({ apdType }) => (
   <div>
     <h2>Proposed Budget</h2>
 
     <h3>Combined Activity Costs</h3>
-    <CombinedActivityCosts isViewOnly />
+    <CombinedActivityCosts isViewOnly apdType={apdType} />
 
     <h3>Summary Budget Table</h3>
-    <BudgetSummary />
+    <BudgetSummary apdType={apdType} />
```

Comment:
**(Non-blocking)** Just a thought- you're pulling in `apdType` from the state, but it's not being used by this component except to pass along as a prop. Instead, you could probably just pull apdType in from the state inside the two components that directly use it.

Reply:
Not sure. If we do that then we have to have more redux-connected components.

Reply:
Hmm, I believe it would be a small addition since both of those components already have a `mapStateToProps` function that pulls in some state data. I _think_ the change would only be that we're adding the apdType to that list instead of it coming in as a passed in prop. At least that's my assumption, but maybe there's something else I didn't consider.

But since this is non-blocking, leaving this way it is ok with me!

Reply:
Good point. I thought that weren't connected yet. I'll leave it for now but me(or someone else) can update it in the future.